app/models/event.rb

```diff
@@ -4,14 +4,15 @@ class Event < ApplicationRecord
 
   PLANNED       = :planned
   REGISTRATION  = :registration
+  LIVE          = :live
```

Comment:
And also will be much easier build query on top of status instead time period...